lib/ReactViews/GyroscopeGuidance/GyroscopeGuidance.jsx

```diff
@@ -134,7 +134,7 @@ export default function GyroscopeGuidance(props) {
       `}
     >
       <MapIconButton
-        // expandInPlace
+        neverCollapse
```

Comment:
I might've missed it, but was there a reason the gyro control buttons are made to never collapse? In a different issue maybe?

Reply:
Alyce Crispy & I workshopped this real quick a while back and decided that for now, this was the best way to resolve the icons jumping around for specifically these controls